azext_iot/central/commands_device_group.py

```diff
@@ -24,3 +26,87 @@ def list_device_groups(
         cmd=cmd, app_id=app_id, token=token, api_version=api_version
     )
     return provider.list_device_groups(central_dns_suffix=central_dns_suffix)
+
+
+def get_device_group(
+    cmd,
+    app_id: str,
+    device_group_id: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+) -> Union[DeviceGroupPreview, DeviceGroupV1_1_preview]:
+    provider = CentralDeviceGroupProvider(
+        cmd=cmd, app_id=app_id, token=token, api_version=api_version
+    )
+    return provider.get_device_group(device_group_id=device_group_id, central_dns_suffix=central_dns_suffix)
+
+
+def create_device_group(
+    cmd,
+    app_id: str,
+    device_group_id: str,
+    content: str,
+    token=None,
+    central_dns_suffix=CENTRAL_ENDPOINT,
+    api_version=ApiVersion.v1_1_preview.value,
+):
+    if not isinstance(content, str):
```

Comment:
the azure cli should technically make this argument into string, especially if you identify this as str in this file (command_device_group)

So there isnt a need for this check.

Reply:
I see. Thanks!